gobcore/events/import_events.py

```diff
@@ -67,13 +67,27 @@ def apply_to(self, entity):
         # Register the application that delivered the event
         entity._application = self._metadata.application
 
-        skip = ["_entity_source_id", "_last_event"]
-
         for key, value in self._data.items():
```

Comment:
This whole method can be rewritten in terms of get_attribute_dict, something like:
for key, value in dict.items:
   setattr(entity, key, value)